components/WorkoutsList/index.js

```diff
@@ -1,24 +1,42 @@
-import Link from "next/link";
 import WorkoutForm from "../WorkoutForm";
-import WorkoutPreview from "../WorkoutPreview";
 import styled from "styled-components";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import Workout from "../Workout";
 
-export default function WorkoutsList({ onAddWorkout, workouts, exercises }) {
+export default function WorkoutsList({
+  onAddWorkout,
+  onDeleteWorkout,
+  workouts,
+  exercises,
+}) {
   return (
     <>
+      <ToastContainer
+        position="top-center"
+        autoClose={3000}
+        hideProgressBar={false}
+        newestOnTop={false}
+        closeOnClick
+        rtl={false}
+        pauseOnFocusLoss
+        draggable
+        pauseOnHover
+        theme="dark"
+      />
       <WorkoutCard>
         <WorkoutForm exercises={exercises} onAddWorkout={onAddWorkout} />
-        {workouts.map((workout) => (
-          <WorkoutItem key={workout.id}>
-            <EditLink href={`workouts/${workout.id}/edit`}>Edit 🖊</EditLink>
-            <WorkoutPreview
-              name={workout.name}
-              workoutExercises={workout.exercises}
-              exercises={exercises}
-              workouts={workouts}
-            />
-          </WorkoutItem>
-        ))}
+        {workouts.length == 0 && (
+          <AlertMessage>
+            Oops! No Workouts yet.<br></br>
+            Create a new Workout to start your journey!
+          </AlertMessage>
+        )}
+        <Workout
+          onDeleteWorkout={onDeleteWorkout}
+          workouts={workouts}
+          exercises={exercises}
+        />
```

Comment:
My guess is that Roland suggested a `</Workout>` component here, which essentially renders the logic you had previously.

Conceptually:
```
{workouts.map((workout) => (
    <Workout key={workout.id} />
))}
```